client/src/pages/recording-view/components/settings-pane.tsx

```diff
@@ -149,7 +149,7 @@ const SettingsPane = () => {
           <ul tabIndex={0} className="dropdown-content menu p-2 shadow bg-base-100 rounded-box w-52">
             {Object.entries(colMaps).map(([value, index]) => (
               <li
-                key={index[0][0]}
+                key={Math.random()}
```

Comment:
this is hacky

Reply:
I knwo but I coundt stand the complains about duplicaterd keys in my console anymore open to hear alternativies.

Reply:
can we not do some form of ++ appending

Reply:
Changed to something that is more reasonable I am acrtually uisngon the colour map name as key